src/plugins/theme/maxHeight.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Max Heights - customized to not produce classes with our custom spacing values
+ */
+module.exports = function () {
+  return {
+    full: '100%',
+    screen: '100vh',
```

Comment:
overriding the TW defaults because the default generated a bunch of classes using our spacing steps (which in our cases don't really make sense as max-heights.